builds/misc/images.yaml

```diff
@@ -32,6 +32,9 @@ jobs:
       - bash: 'cargo install --git https://github.com/arsing/cross.git --branch set-path'
         displayName: 'Install cross (fork with docker fix)'
 
+      - bash: 'docker system prune -af --volumes'
```

Comment:
you should not need this as this is MS hosted agent; it should be clean image every time starting a new job.

Reply:
That's not the behavior that I was observing.

Reply:
Please confirm with @philipktlin that this is okay.

Reply:
let's chat tomorrow.

Reply:
This is used to remove cached container images in MS hosted agent to free up more spaces.